truss/tests/templates/server/test_schema.py

```diff
@@ -0,0 +1,187 @@
+import inspect
+from typing import AsyncGenerator, Awaitable, Generator, Union
+
+from pydantic import BaseModel
+from truss.templates.server.common.schema import TrussSchema
+
+
+class ModelInput(BaseModel):
+    input: str
+    stream: bool
+
+
+class ModelOutput(BaseModel):
+    output: str
+
+
+def test_truss_schema_pydantic_input_and_output():
+    def predict(request: ModelInput) -> ModelOutput:
```

Comment:
when writing truss code, the code will look like 
```
class Model:
    def predict(self, request: ModelInput) -> ModelOutput:
```
I think it'd be great if this test reflected the same. It's not obvious to me that adding self would produce the same results as this.